todo_app/data/session_items.py

```diff
@@ -44,30 +73,49 @@ def add_item(title):
     items = get_items()
 
     # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
-
-    return item
-
+    id = items[-1].id + 1 if items else 0
 
-def save_item(item):
+    # Obtain todo_id
+    r=requests.get(base_url + 'boards/' + id_board + '/lists' , params = payload)
+    r=r.json()
+    
+    for list in r:
+        if 'To Do' == list['name']:
+            todo_id = list['id']
+            break
+
+    # Create item card in Trello using to Do list as default
+    r=requests.post(base_url+'cards?'+'idList='+todo_id+'&name='+title , params = payload)
+    r=r.json()
+    # Obtain trello item id
+    id_card = r['id']
+    # Obtain trello list id
+    id_list = r['idList']
+    item = Item(id,id_card,'To do',id_list, title)
+    return 
```

Comment:
This is unnecessary - you aren't doing anything with `item` and an empty `return` statement at the end of the function is the same as no `return` statement.